src/Children.js

```diff
@@ -0,0 +1,116 @@
+import * as Constant from './Constant.js'
+import * as Utility from './Utility.js'
+import * as Element from './Element.js'
+
+export default {only: only, count: count, filter: filter, find: find, map: map, forEach: each, toArray: array}
```

Comment:
the `{forEach: each}` (and `toArray: array`) rename is confusing, change the function names to match